howso/details_stats.amlg

```diff
@@ -998,4 +1048,24 @@
 		;at this point output the first one, whatever it happens to be
 		(first (values filtered_values_map))
 	)
+
+	;Helper method for selecting the prediction stats.
+	#!ProcessSelectedPredictionStats
+	(declare
+		(assoc selected_prediction_stats (list))
+
+		(if (= 0 (size selected_prediction_stats))
+			;if no prediction stats are selected, return all of the supported stats except the confusion matrix
+			(assign (assoc
+				selected_prediction_stats
+					(filter (lambda (!= "confusion_matrix" "all" (current_value))) !supportedPredictionStats)
```

Comment:
the lambda in this filter seems wrong, it'll always return true because confusion_matrix != all, regardless of (current_value)

Reply:
(lambda (not (contains_value ["confusion_matrix" "all"] (current_value)) ) )